Sources/Transaction/Inputs/DefaultMixinSelectionStrategy.swift

```diff
@@ -4,16 +4,21 @@
 
 import Foundation
 
-final class DefaultMixinSelectionStrategy: MixinSelectionStrategy {
+final class DefaultMixinSelectionStrategy: IdempotentMixinSelectionStragegy {
     let offsetParam: UInt64 = 88
-    var rng: RandomNumberGenerator = SystemRandomNumberGenerator()
+    var rng: MobileCoinRng
+
+    init(rng: MobileCoinRng = MobileCoinDefaultRng()) {
```

Comment:
do we need this on the init if the function calls require it to be passed in ?

Reply:
Left over from earlier code pass where I had it as an attribute... nice catch.

Reply:
code updated